test/cli/test-fail/missing-resource/test.yaml

```diff
@@ -1,11 +1,15 @@
-name: test-simple
-policies:
+apiVersion: cli.kyverno.io/v1alpha1
+kind: PolicyTests
+metadata:
+ name: test-simple
+spec:
+ policies:
   - policy.yaml
-resources:
+ resources:
   - resources.yaml
-results:
+ results:
   - policy: disallow-latest-tag
     rule: validate-image-tag
     resource: missing
     kind: Pod
-    result: pass
+    result: pass
```

Comment:
Newlines are missing at the end of files quite a bit here.

Reply:
Resolved this in the latest commit.